cli/js/files.ts

```diff
@@ -114,9 +114,66 @@ export class File
   }
 }
 
-export const stdin = new File(0);
-export const stdout = new File(1);
-export const stderr = new File(2);
+class Stdin implements Reader, SyncReader, Closer {
+  readonly rid: number;
+  constructor() {
+    this.rid = 0;
+  }
+
```

Comment:
Those classes are completely internal - public types export intersection of `Reader`, `SyncReader` and `Closer`

Reply:
Actually I exported interfaces: `Stdin`, `Stdout` and `Stderr` - this allows for a hack for power users we discussed offline.